engine/src/test/java/com/redhat/cloud/notifications/processors/camel/teams/TeamsProcessorTest.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.cloud.notifications.processors.camel.teams;
+
+import com.redhat.cloud.notifications.TestLifecycleManager;
+import com.redhat.cloud.notifications.processors.camel.CamelProcessor;
+import com.redhat.cloud.notifications.processors.camel.CamelProcessorTest;
+import com.redhat.cloud.notifications.processors.camel.InternalCamelTemporaryService;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import javax.inject.Inject;
+
+
+@QuarkusTest
+@QuarkusTestResource(TestLifecycleManager.class)
+public class TeamsProcessorTest extends CamelProcessorTest {
+
+    private static final String GOOGLE_SPACES_TEMPLATE = "{#if data.context.display_name??}" +
```

Comment:
```suggestion
    private static final String TEAMS_TEMPLATE = "{#if data.context.display_name??}" +
```

Reply:
We should use the Teams template here instead of the Slack one. But first, I need to update the Teams template 😅